scanpy/tools/_palantir.py

```diff
@@ -0,0 +1,299 @@
+"""Run Diffusion maps using the adaptive anisotropic kernel
+"""
+from .. import logging as logg
+
+def palantir( adata ):
+
+    """
+    Run Diffusion maps using the adaptive anisotropic kernel [Setty27]_.
+
+    Palantir is an algorithm to align cells along differentiation trajectories.
+    Palantir models differentiation as a stochastic process where stem cells
+    differentiate to terminally differentiated cells by a series of steps through
+    a low dimensional phenotypic manifold. Palantir effectively captures the
+    continuity in cell states and the stochasticity in cell fate determination.
+    Palantir has been designed to work with multidimensional single cell data
+    from diverse technologies such as Mass cytometry and single cell RNA-seq.
+
+    Full documentation can be found here https://github.com/dpeerlab/Palantir
```

Comment:
Could you make this a `.. note:` as in https://github.com/theislab/scanpy/blob/master/scanpy/preprocessing/_dca.py. Using the same wording: `More information and bug reports ...`.